website/docs/d/public_ips.html.markdown

```diff
@@ -22,7 +22,8 @@ data "azurerm_public_ips" "example" {
 ## Argument Reference
 
 * `resource_group_name` - Specifies the name of the resource group.
-* `attached` - (Optional) Filter to include IP Addresses which are attached to a device, such as a VM/LB (`true`) or unattached (`false`).
+* `attached` - (Optional) Filter to include IP Addresses which are attached to a device, such as a VM/LB (`true`) or unattached (`false`). This option is deprecated for `attachment_status`. Default value is `false`, unless `attachment_status` is set.
```

Comment:
We can remove the doc entries for deprecated properties, this helps move users over more smoothly as new consumers of the resource will typically start using the replacement properties.
```suggestion
```

Reply:
Done ✅